statsd/config.go

```diff
@@ -0,0 +1,44 @@
+package statsd
```

Comment:
The config is not used anywhere else. Each object has its own config so it seems over-engineered.